aces/Loris
# [tools] Move and Clean-up of MyISAM to INNODB scripts

This PR moves the `Engine_Change_MyISAM_to_INNODB_schemaTables.php` and `Engine_Change_MyISAM_to_INNODB_allTables.php` to the new tools subdirectory and combines them to reduce code duplication. 

A new feature was also added to run the change on a single table in the DB 

This PR also cleans up the scripts and adds an automatic datacorruption validation check, it compares the values before and after the conversion of the tables and makes sure the data is the same.

A backup of the tables is still highly recommended before running either of these scripts


Testing instructions:
- make sure you have MyISAM tables in the DB (if not the script will throw an error and die)
- run the script using the `tosql` option with each of the possible actions `schemaTables`, `allTables` and/or a specific table name for any table that has a MyISAM engine. Check the output SQL after each run to make sure the SQL commands are correct. (don't run the SQL commands on the DB if you do not wish to reset the database before the next step)
- run the script using the `confirm` option with each of the possible actions (this will generate the SQL and run it so this will make changes to the DB for sure. I suggest starting by testing the single table action, then the schema then the all this way you can test all options without ever needing to reset the DB.
- add the Pass manual tests flag

Notice: possible memory issues if the table being converted is too large.
